src/index.html

```diff
@@ -3,12 +3,66 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="menu">
+          <li>
+            <a class="link is-active" href="#">
+              Apple
+            </a>
+          </li>
+          <li>
```

Comment:
![image](https://user-images.githubusercontent.com/98764641/202693834-2f512b14-b929-445f-a0e8-0133046a00bc.png)
check it everywhere